backend/module/eCampCore/src/Hydrator/PeriodHydrator.php

```diff
@@ -34,8 +35,8 @@ public function extract($object) {
         return [
             'id' => $period->getId(),
             'description' => $period->getDescription(),
-            'start' => Util::extractDateTime($period->getStart()),
-            'end' => Util::extractDateTime($period->getEnd()),
+            'start' => $period->getStart()->__toString(),
+            'end' => $period->getEnd()->__toString(),
```

Comment:
`__toString()` müsst doch automatisch aufgerufen werden, oder?

Reply:
Leider nein, das spielt wohl eine Rolle was der Hydrator extract macht

Reply:
Nach einiges an Debug-stack-trace: Am Ende ist es kein Hydrator oder Laminas-Komponente, welches ein Mapping vornimmt sondern einfach ein pures `json_encode` :dizzy_face:

Und das Verhalten kann man überschreiben, siehe hier:
https://stackoverflow.com/questions/40282868/change-output-of-datetime-in-json-encode